src/genie/libs/parser/iosxe/show_snmp.py

```diff
@@ -79,3 +79,118 @@ def cli(self, output=None):
 
         return ret_dict
 
+
+
```

Comment:
Need to remove extra space from top.
Also need to add comment for all new parsers we add

Reply:
Removed extra lines. I also added comments. If I am still missing a comment please let me know and I can update.